### ISSUE TITLE
"Error: Unknown error" when trying to login to self-hosted standalone server

### ISSUE THREAD

**Issue by bigjdunham** [standardnotes/web#824]
I get the dreaded "Error: Unknown error." message when I try to login to my account hosted on my standalone Standard Notes server. I can login just fine from the Linux Desktop app, from the iPad OS app, and the Android app. It's just when I try to login from the web client, either self-hosted or at app.standardnotes.com. I cannot for the life of me find a more descriptive reason in either the logs of the web client or the logs of the server, as to why this happens. Any help would be much appreciated. I am using the latest Standard Notes Web app via Docker, and the latest Standalone Server via Docker. Thanks.

**Comment by bigjdunham** [comment]
I can add that I have also attempted to create new accounts with the self-hosted server from the web client and have been unable to do so, getting the same "Error: Unknown error." message.

**Comment by moughxyz** [comment]
Is your server behind https? That is a requirement in order for you to be able to access it from our web app.

**Comment by bigjdunham** [comment]
Yes it's behind an https proxy server. And my Android, iPad, and Linux Desktop clients have no trouble accessing it through the https proxy. It's just the web app.

**Comment by moughxyz** [comment]
Try opening the Network tab in the dev console and see what the http status code is for the failed request.

**Comment by bigjdunham** [comment]
Okay finally addressing this again. Sorry my delay. Opening the Network tab on the Dev console gives me these two errors:
CORS Missing Allow Origin
NS_ERROR_DOM_BAD_URI

**Comment by moughxyz** [comment]
You'll need to return the appropriate headers in your server configuration to allows CORS requests.
